src/master-playlist-controller.js

```diff
@@ -66,36 +67,6 @@ const objectChanged = function(a, b) {
 };
 
 /**
- * Parses a codec string to retrieve the number of codecs specified,
```

Comment:
This function doesn't seem to be reliant on being here (there are a few other functions here that could be moves as well).  I shifted this into a util module as I wanted to use this to check whether a playlist contained video or not in the playlist selection algorithm below.